recipes/r-lasr/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "lasr" %}
+{% set version = "0.13.7" %}
+{% set posix = "m2-" if win else "" %}
+
+package:
+  name: r-{{ name }}
+  version: '{{ version }}'
+
+source:
+  url: https://r-lidar.r-universe.dev/src/contrib/lasR_{{ version }}.tar.gz
+  sha256: 746e84837af8a2d294321b76c790be5f0d57fc9e305a2e0f72703926a68f64b7
+
+build:
+  skip: true  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  
+
+requirements:
+  build:
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+    - r-rcpp                       # [build_platform != target_platform]
+    - {{ compiler('c') }}          # [not win]
+    - {{ stdlib('c') }}            # [not win]
+    - {{ compiler('m2w64_c') }}    # [win]
+    - {{ stdlib('m2w64_c') }}      # [win]
+    - {{ compiler('cxx') }}        # [not win]
+    - {{ compiler('m2w64_cxx') }}  # [win]
+    - {{ posix }}autoconf
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}make
+    - pkg-config
+    - posix                        # [win]
+    - llvm-openmp                  # [osx]
+  host:
+    - r-base
+    - r-rcpp
+    - r-r.utils
+    - geos
```

Comment:
This lib is already ensured by `libgdal-core` and linking analysis indicates that this package does not directly link.

```suggestion
```